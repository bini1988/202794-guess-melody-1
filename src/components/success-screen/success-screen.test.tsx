import React from "react";
import renderer from "react-test-renderer";
import SuccessScreen from "./success-screen";


it(`SuccessScreen correctly renders default markup`, () => {
  const tree = renderer
    .create(
        <SuccessScreen
          onRestart={() => {}}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
