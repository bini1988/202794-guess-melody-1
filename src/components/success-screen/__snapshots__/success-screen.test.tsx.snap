// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessScreen correctly renders default markup 1`] = `
<section
  className="result"
>
  <div
    className="result__logo"
  >
    <img
      alt="Угадай мелодию"
      height="83"
      src="img/melody-logo.png"
      width="186"
    />
  </div>
  <h2
    className="result__title"
  >
    Вы настоящий меломан!
  </h2>
  <p
    className="result__total"
  >
    За 3 минуты и 25 секунд вы набрали 12 баллов (8 быстрых), совершив 3 ошибки
  </p>
  <p
    className="result__text"
  >
    Вы заняли 2 место из 10. Это лучше чем у 80% игроков
  </p>
  <button
    className="replay"
    onClick={[Function]}
    type="button"
  >
    Сыграть ещё раз
  </button>
</section>
`;
